platform/macos/src/MGLMapView.h

```diff
@@ -47,6 +47,7 @@ NS_ASSUME_NONNULL_BEGIN
  @note You are responsible for getting permission to use the map data and for
     ensuring that your use adheres to the relevant terms of use.
  */
+MGL_EXTERN
```

Comment:
Putting `MGL_EXTERN` keyword on a separate line before the `IB_DESIGNABLE` keyword seems to break Quick Help and the index in general in Xcode 8.2. If you ⌥-click a member of a class or open the code completion popup, you get the documentation about the class itself rather than the member. If you ⌘-click to jump to the definition of a member, you get the line above it instead.

release-ios-v3.4.0 [on this line](https://github.com/mapbox/mapbox-gl-native/blob/76925d25788f163726f5897e2255633fe3fcdc90/platform/macos/app/MapDocument.m#L660):
<img width="559" alt="release-quick-help" src="https://cloud.githubusercontent.com/assets/1231218/21166186/ca31c320-c158-11e6-848f-a3201d690d0e.png"><br>
![release-code-completion](https://cloud.githubusercontent.com/assets/1231218/21166261/25b06918-c159-11e6-9e9d-337faf92d560.png)

macos-sdk-symbol-visibility [on the same line](https://github.com/mapbox/mapbox-gl-native/blob/76925d25788f163726f5897e2255633fe3fcdc90/platform/macos/app/MapDocument.m#L660):
<img width="572" alt="pr-quick-help" src="https://cloud.githubusercontent.com/assets/1231218/21166194/d285878c-c158-11e6-8584-ae68405223e2.png"><br>
![pr-code-completion](https://cloud.githubusercontent.com/assets/1231218/21166284/5553cb9c-c159-11e6-91c9-7d8d212cd62d.png)

Putting the two keywords on the same line seems to fix the problem, but I’ve only tried this so far inside macos.xcodeproj so far.